README.md

```diff
@@ -17,7 +17,7 @@ endpoint: AAD v2.0
 
 This sample demonstrates how to use MSAL.NET to:
 
-- authenticate the user silently using username and password
+- authenticate the user silently using username and password. [Resource owner password credentials](https://tools.ietf.org/html/rfc6749#section-1.3.3)
```

Comment:
Strictly speaking, username password is only ROPC for managed users. For federated users it's not ROPC. I think we'd want to move this link to the bottom of the README.md (more info)?

Reply:
thanks for clarifying..